### DISCUSSION TITLE
WSL 中 VSCode Server 下载 remote 的时候出现 Temporary failure in name resolution

### DISCUSSION THREAD

**Discussion by jiegec** [physics-data/faq#29] [Q&A] [answered]
现象如下：

![image](https://user-images.githubusercontent.com/6127678/129471811-9859b8d6-0810-4460-b951-8c7e8ab147c7.png)

Windows 版本：1709（Build 16299）

**Comment by jiegec** [comment, marked as answer]
可以参考的一些解决办法：https://github.com/microsoft/WSL/issues/5256

**Comment by heroxbd** [comment]
FYI:  vscode 还没有官方的 Debian 包 https://bugs.debian.org/898259

**Reply by jiegec** [reply]
用这个：https://code.visualstudio.com/docs/setup/linux#_debian-and-ubuntu-based-distributions

**Reply by Harry-Chen** [reply]
> FYI: vscode 还没有官方的 Debian 包 https://bugs.debian.org/898259

这是 vscode server
